app/views/exam_templates/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_for @exam_templates,
```

Comment:
I think you should use the `exam_template` local variable (you pass this to the partial)